projects/python/perception/object_detection_2d/nanodet/eval_demo.py

```diff
@@ -20,15 +20,16 @@
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
+    parser.add_argument("--dataset", help="Dataset to evaluate on", type=str, default="coco", choices=["voc", "coco"])
     parser.add_argument("--data-root", help="Dataset root folder", type=str)
-    parser.add_argument("--model", help="Model that config file will be used", type=str)
+    parser.add_argument("--model", help="Model that config file will be used", type=str, default="m")
```

Comment:
Wording is somewhat confusing.
```suggestion
    parser.add_argument("--model", help="Model for which a config file will be used", type=str, default="m")
```